fe1-web/src/features/evoting/screens/CreateElection.tsx

```diff
@@ -133,34 +157,77 @@ const CreateElection = ({ route }: any) => {
           setElectionName(text);
         }}
       />
-      {/* see archive branches for date picker used for native apps */}
-      {Platform.OS === 'web' && buildDatePickerWeb()}
-      {questions.map((value, idx) => (
-        <View key={idx.toString()}>
-          <TextInputLine
-            placeholder={STRINGS.election_create_question}
-            onChangeText={(text: string) =>
-              setQuestions((prev) =>
-                prev.map((item, id) => (id === idx ? { ...item, question: text } : item)),
-              )
-            }
-          />
-          <TextInputList
-            onChange={(ballot_options: string[]) =>
-              setQuestions((prev) =>
-                prev.map((item, id) =>
-                  id === idx ? { ...item, ballot_options: ballot_options } : item,
-                ),
-              )
-            }
-          />
-        </View>
-      ))}
+      <DropdownSelector
+        selected={version}
+        onChange={(value) => {
+          if (value) {
+            setVersion(value as ElectionVersion);
+          }
+        }}
+        options={[
+          {
+            value: ElectionVersion.OPEN_BALLOT,
+            label: STRINGS.election_create_version_open_ballot,
+          },
+          {
+            value: ElectionVersion.SECRET_BALLOT,
+            label: STRINGS.election_create_version_secret_ballot,
+          },
+        ]}
+      />
+      {
+        // the date picker for the web
+        // see archive branches for date picker used for native apps
+        Platform.OS === 'web' && (
+          <View style={styles.viewVertical}>
+            <View style={[styles.view, { padding: 5 }]}>
+              <ParagraphBlock text={STRINGS.election_create_start_time} />
+              <DatePicker
+                selected={startTime.toDate()}
+                onChange={(date: Date) =>
+                  onChangeStartTime(date, setStartTime, setEndTime, DEFAULT_ELECTION_DURATION)
+                }
+              />
+            </View>
+            <View style={[styles.view, { padding: 5, zIndex: 'initial' }]}>
```

Comment:
since you're sort-of changing this file, I'd try to address the lint warnings about inline styles 😉